app.js

```diff
@@ -5,15 +5,27 @@ const session = require("express-session");
 const dotenv = require("dotenv");
 const MongoDBStore = require("connect-mongodb-session")(session);
 const flash = require("connect-flash");
+const cookieParser = require("cookie-parser");
+const csrf = require("csurf");
```

Comment:
Believe you’re supposed to use host-csrf since csurf is deprecated, but did you hear otherwise from the mentor sessions?